### ISSUE TITLE
win_copy: Unable to copy large folder

### ISSUE THREAD

**Issue by Udayendu** [ansible-collections/community.windows#138]
##### SUMMARY
**win_copy** is unable to copy large folder. Max it can copy a file/directory of 2.5GiB in size. 

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
- win_copy

##### ANSIBLE VERSION
```
# pip3 show ansible
Name: ansible
Version: 2.9.9
Summary: Radically simple IT automation
Home-page: https://ansible.com/
Author: Ansible, Inc.
Author-email: info@ansible.com
License: GPLv3+
Location: /usr/local/lib/python3.8/dist-packages
Requires: jinja2, cryptography, PyYAML
Required-by:
```

##### CONFIGURATION
- No specific configuration

##### OS / ENVIRONMENT
- Ubuntu 20.04 LTS Server

##### STEPS TO REPRODUCE
- Use the below playbook to copy a folder of 4.5 GiB in size from ansible provision server(ubuntu server) to a windows 2016 server, it will fail.
```
---
- name: Copy SQL files to Windows VM
  hosts: "{{ target }}"
  gather_facts: false
  connection: winrm

  tasks:
    - name: Copy SQL Files using win_copy module
      win_copy:
        src: /root/software/
        dest: C:\SQL2016_standard\
```

```
# tree /root/software/
/root/software/
└── SQL2016_standard
    ├── SQL2016Standardw_SP1.zip
    ├── SSMS2016_setup
    │   └── SSMS-Setup-ENU.exe
    └── updates
        ├── SQLServer2016-KB4524334-x64.exe
        └── SQLServer2016SP2-KB4052908-x64-ENU.exe
```
```
# cat inventory
[mssql]
xxx.xxx.xxx.xxx

[mssql:vars]
ansible_connection=winrm
ansible_user=global\myadmin
ansible_password=xxxxxxxxxxxxx
ansible_winrm_transport=ntlm
ansible_winrm_server_cert_validation=ignore
ansible_port=5985
```

# Command to deploy
```
# ansible-playbook -i  inventory -e target=mssql wincp.yaml -vvv
```

##### EXPECTED RESULTS
It should copy the folder successfully.

##### ACTUAL RESULTS
Its failing with the following traces:

```
# ansible-playbook -i  inventory -e target=mssql-sentry-sql wincp.yaml -vvv
ansible-playbook 2.9.9
  config file = None
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.8/dist-packages/ansible
  executable location = /usr/local/bin/ansible-playbook
  python version = 3.8.2 (default, Apr 27 2020, 15:53:34) [GCC 9.3.0]
No config file found; using defaults
host_list declined parsing /root/playbooks/inventory as it did not pass its verify_file() method
script declined parsing /root/playbooks/inventory as it did not pass its verify_file() method
auto declined parsing /root/playbooks/inventory as it did not pass its verify_file() method
[DEPRECATION WARNING]: The TRANSFORM_INVALID_GROUP_CHARS settings is set to allow bad characters in group names by default, this will change, but still be user configurable on deprecation. This feature will be removed in version 2.10. Deprecation
warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.
[WARNING]: Invalid characters were found in group names but not replaced, use -vvvv to see details
Parsed /root/playbooks/inventory inventory source with ini plugin

PLAYBOOK: wincp.yaml *****************************************************************************************************************************************************************************************************************************************
1 plays in wincp.yaml

PLAY [Copy SQL files to Windows VM] **************************************************************************************************************************************************************************************************************************
META: ran handlers

TASK [Copy SQL Files using win_copy module] ******************************************************************************************************************************************************************************************************************
task path: /root/playbooks/wincp.yaml:8
Using module file /usr/local/lib/python3.8/dist-packages/ansible/modules/windows/win_copy.ps1
Pipelining is enabled.
<xxx.xxx.xxx.xxx> ESTABLISH WINRM CONNECTION FOR USER: global\myadmin on PORT 5985 TO xxx.xxx.xxx.xxx
EXEC (via pipeline wrapper)
EXEC (via pipeline wrapper)
<xxx.xxx.xxx.xxx> PUT "/root/.ansible/tmp/ansible-local-1581142w2j6iv0m/tmphitjyjsd/win_copy.zip" TO "C:\Users\myadmin\AppData\Local\Temp\ansible-tmp-1598965263.1938927-1581146-49202896425265\source.zip"
EXEC (via pipeline wrapper)
The full traceback is:
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/winrm/protocol.py", line 248, in send_message
    root = ET.fromstring(ex.response_text)
  File "/usr/lib/python3.8/xml/etree/ElementTree.py", line 1321, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/ansible/executor/task_executor.py", line 146, in run
    res = self._execute()
  File "/usr/local/lib/python3.8/dist-packages/ansible/executor/task_executor.py", line 653, in _execute
    self._handler.cleanup()
  File "/usr/local/lib/python3.8/dist-packages/ansible/plugins/action/__init__.py", line 128, in cleanup
    self._remove_tmp_path(self._connection._shell.tmpdir)
  File "/usr/local/lib/python3.8/dist-packages/ansible/plugins/action/__init__.py", line 407, in _remove_tmp_path
    tmp_rm_res = self._low_level_execute_command(cmd, sudoable=False)
  File "/usr/local/lib/python3.8/dist-packages/ansible/plugins/action/__init__.py", line 1075, in _low_level_execute_command
    rc, stdout, stderr = self._connection.exec_command(cmd, in_data=in_data, sudoable=sudoable)
  File "/usr/local/lib/python3.8/dist-packages/ansible/plugins/connection/winrm.py", line 548, in exec_command
    result = self._winrm_exec(cmd_parts[0], cmd_parts[1:], from_exec=True, stdin_iterator=stdin_iterator)
  File "/usr/local/lib/python3.8/dist-packages/ansible/plugins/connection/winrm.py", line 460, in _winrm_exec
    command_id = self.protocol.run_command(self.shell_id, to_bytes(command), map(to_bytes, args), console_mode_stdin=(stdin_iterator is None))
  File "/usr/local/lib/python3.8/dist-packages/winrm/protocol.py", line 359, in run_command
    res = self.send_message(xmltodict.unparse(req))
  File "/usr/local/lib/python3.8/dist-packages/winrm/protocol.py", line 251, in send_message
    raise ex
  File "/usr/local/lib/python3.8/dist-packages/winrm/protocol.py", line 243, in send_message
    resp = self.transport.send_message(message)
  File "/usr/local/lib/python3.8/dist-packages/winrm/transport.py", line 323, in send_message
    response = self._send_message_request(prepared_request, message)
  File "/usr/local/lib/python3.8/dist-packages/winrm/transport.py", line 339, in _send_message_request
    raise WinRMTransportError('http', ex.response.status_code, response_text)
winrm.exceptions.WinRMTransportError: Bad HTTP response returned from server. Code 400
fatal: [100.89.14.149]: FAILED! => {
    "msg": "Unexpected failure during module execution.",
    "stdout": ""
}

PLAY RECAP ***************************************************************************************************************************************************************************************************************************************************
xxx.xxx.xxx.xxx              : ok=0    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0

```

**Comment by Udayendu** [comment]
I tried to copy a single file of 6.7GiB and that worked. But when keeping that in a folder its failing with the same error.

**Comment by Udayendu** [comment]
Duplicate of https://github.com/ansible-collections/ansible.windows/issues/97
